tests/test_endtoend.py

```diff
@@ -148,6 +150,90 @@ def f(x, y):
     _assert_pytree_isequal(vjp, vjp_raw)
 
 
+@pytest.mark.parametrize("use_jit", [True, False])
+@pytest.mark.parametrize(
+    "jacfun", [partial(jax.jacfwd, argnums=(0, 1)), partial(jax.jacrev, argnums=(0, 1))]
```

Comment:
I suggest we use strings to distinguish between the cases, for better rendering when running the tests and to decouple the logic here a bit.